README.md

```diff
@@ -61,14 +61,12 @@ installed system wide. To do this:
    programmable completion for other purposes.
 
 ### macOS (OS X)
-
-If you're using macOS (formerly OS X), `/etc/bashrc` is apparently not sourced at
```

Comment:
I have again looked at the "Installation" section of README. The reason that we had a separate section for macOS seems to be because `/etc/bashrc` wasn't sourced then, but is it still the case? If yes, I think we should still want this sentence.

Reply:
Added

Reply:
`/etc/bashrc` and `~/.bashrc` are different files, but yeah, `~/.bashrc` is not configured in `~/.bash_profile` probably because they switched to Zsh and doesn't provide `~/.bash_profile` by default.

Reply:
I'll add a description.

Reply:
The convention is `/etc/bashrc` is system wide and `~/.bashrc` is per-user. Neither are sourced by default by MacOS, nor is it common to edit the systemwide login file either.

Reply:
I added 3d50f47179ed3aaf3aa20f85ead9eda5591b80c0. Actually, `~/.bash_profile` is not really a proper file to write interactive settings. I originally thought of discussing this in a separate issue, but the present discussion of `/etc/bashrc` and `~/.bashrc` is closely related to this, so I decided to include it in this PR.

There are additional style fixes d4c5fea5418a9729dcdb49fd1907635ad699ca03 and 0a11552a757a350013dc21bbadeff51bcbe99daf. I'll squash these small fixes later.

Reply:
> nor is it common to edit the systemwide login file either.

Right, but `bash-completion`'s README is not written specifically for normal users. This README also targets package maintainers of different systems.

Reply:
I added another commit 78ae7f6a689fed77d1c7bf35ea50f3f95669c10b to clarify that the user needs to install the `bash-completion` package using the Homebrew. To the users unfamiliar with it, the current wording appears to imply that Homebrew is provided by macOS by default, and the `bash-completion` package is also installed by default.

Actually, the reason that we had a special section for macOS was that `/etc/bashrc` and `~/.bashrc` didn't work properly. For the other environments where `/etc/bashrc` (or a similar file) and `~/.bashrc` work, we haven't described the settings for specific package managers. In that sense, it might be a bit strange that we describe the setting for Homebrew. The setting for Homebrew's `bash-completion` package seems to be anyway presented to the user on the installation [1], so it is technically unnecessary.

- [1] https://github.com/Homebrew/homebrew-core/blob/5df158e314883cda5ff0f9c258d289af64c712db/Formula/b/bash-completion%402.rb#L49-L54

Nevertheless, I think it would be useful for macOS users to provide the Homebrew setting as an example.